Makefile

```diff
@@ -1,34 +1,18 @@
-# Copyright 2015-2019 Capital One Services, LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
 COLOR ?= always # Valid COLOR options: {always, auto, never}
 CARGO = cargo --color $(COLOR)
 TARGET = target/wasm32-unknown-unknown
 DEBUG = $(TARGET)/debug
 RELEASE = $(TARGET)/release
 KEYDIR ?= .keys
 
-.PHONY: all bench build check clean doc test update keys keys-account keys-module
+.PHONY: all build check clean doc test update
 
 all: build
 
-bench:
-	@$(CARGO) bench
-
 build:
 	@$(CARGO) build
-	wascap sign $(DEBUG)/{{crate_name}}.wasm $(DEBUG)/{{crate_name}}_signed.wasm -i $(KEYDIR)/account.nk -u $(KEYDIR)/module.nk -s -n {{crate_name}}
+	VERSION=$(cargo metadata --no-deps --format-version 1 | jq -r '.packages[].version')
+    wash claims sign $(DEBUG)/{{crate_name}}.wasm -c wasmcloud:httpserver --name "{{crate_name}}" --ver $VERSION --rev 0
```

Comment:
Make sure this doesn't have spaces, `wash` needs to be preceded by a tab.

Also, if you decide to move the `VERSION` variable to the top-level section, change `$VERSION` to `$(VERSION)`